tests/Microsoft.Identity.Test.Performance/AcquireTokenForClientLargeCacheTests.cs

```diff
@@ -78,11 +90,20 @@ public async Task AcquireTokenForClient_DifferentScopes_TestAsync()
 
             await _ccaTokensDifferByScope.AcquireTokenForClient(new[] { scope })
               .WithForceRefresh(false)
-              .WithAuthority($"https://login.microsoft.com/tid")
+              .WithAuthority($"https://login.microsoftonline.com/tid")
               .ExecuteAsync()
               .ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// This is a test to validate the performance impact of calculating the TokenCacheNotificationArgs.SuggestedCacheExpiry when saving an access token.
+        /// </summary>
+        [Benchmark(Description = "Token cache expiry - O(n)")]
+        public async Task CaclulateTokenCacheExpiryTestTestAsync()
+        {
+            await _ccaTokensDifferByScope.AppTokenCacheInternal.SaveTokenResponseAsync(_requestParams, _response).ConfigureAwait(false);
```

Comment:
Was under the impression that this test is not good enough because `SaveTokenResponseAsync` executes in under 100ms, and so the measurements are not reliable.

Reply:
Don't we expect the cache to be small, since we partition by tenant Id / client Id. We could just add extra sizes to `TokenCacheSize` (10k, 100k).

Reply:
Worth a try. I remember that when I tried to populate a very large cache (100k), the operation did not finish in the 1h I had dinner :).

Reply:
@trwalke - since test is inconclusive, please delete it.